manifests/cluster-authentication-operator_03_version-mapping.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-authentication-operator
+  name: version-mapping
```

Comment:
Do we need this?  I think it is only used in the `VersionForOperand` code.

Reply:
I couldn't find `version-mapping` code, was looking at bad places (CVO repo) :man_facepalming: I'll use this instead of the `IMAGE_VERSION` env var in the payload deployment.